CMakeLists.txt

```diff
@@ -177,7 +175,7 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU" OR
 		set(GCC_EXTRA_OPTIONS "${GCC_EXTRA_OPTIONS} ${FLAG_TESTED}")
 	endif()
 	#
-	set(yaml_cxx_flags "-Wall ${GCC_EXTRA_OPTIONS} -pedantic -Wno-long-long ${yaml_cxx_flags}")
+	set(yaml_cxx_flags "-Wall ${GCC_EXTRA_OPTIONS} -pedantic -Wno-long-long -std=c++11 ${yaml_cxx_flags}")
```

Comment:
I believe it's safe to stack those flags, `-std=c++11 -std=c++14` is treated the same as `std=c++14` (at least in Clang)